esp-backtrace/src/lib.rs

```diff
@@ -259,83 +196,85 @@ fn is_valid_ram_address(address: u32) -> bool {
     true
 }
 
-#[cfg(all(
-    any(
-        not(any(feature = "esp32", feature = "esp32p4", feature = "esp32s3")),
-        not(feature = "halt-cores")
-    ),
-    not(feature = "custom-halt")
-))]
 #[allow(unused)]
 fn halt() -> ! {
-    loop {
-        continue;
+    cfg_if::cfg_if! {
+        if #[cfg(feature = "custom-halt")] {
+            // call custom code
+            extern "Rust" {
+                fn custom_halt() -> !;
+            }
+            unsafe { custom_halt() }
+        } else if #[cfg(any(feature = "esp32", /*feature = "esp32p4",*/ feature = "esp32s3"))] {
```

Comment:
Is the P4 commented out intentional here?

Reply:
Yes